concepts/observability-strategy/configurable-monitoring/assets/monitoring-arch.drawio.svg

```diff
@@ -1,14 +1,14 @@
bM2jYcBQ/3jZMkqneBqrpa3gTbUT5ejdIbzrdMVPwXKiR1seTQbxUXLJ5BXdwlKBM9KXOJQYNCmGaPJIAA6vHeA68LADN67QMJ7jeliOzq8hybwXmcXfuD9B95/4P0H3v9geB9APAhcHd4PnDEofFgDeF8FPCovCqqAXw1qAr5rAvC1XhRU6FjDkh6UakgqKS3BUUnfGooYEAnUL+GivLyEinLJVyhRdz1t+I8SHdgEJS7Qq3zYq2GhMbS6UA5XlFinYTVDuR2sLtcTU58sOlQuih0UtgflwAKMdklJhTJCfEvD8OJShinVAI4m1cxsnXLhsz1R+AZSJGnfcCiJWnn/WvCqFzleFr33IosXFyn/FkSKRy06lSn/vchUhW9NSPsmINp14hu0b0EYHX/PeBvsGt+N8NrcxBBcNLSgxgyziZq/fhlRx+kJL6NksyjsW9Y7E8byX8U145QbVEmOoz5x8CI8yanxbzJFsF8mpRSBybCqI6UIbGCrbjbQWF1eV0aXrWYI7pfLKJyQ9ZHERknfIpLRIen7ksULNGkDW5ec6cze5aqzQfovmwUiLS+bLMcLdV38mhGmkHs9vxE6PKwyutSXxO21HihBzS6UFnmGTrkFxYXiarilyz90t1AGKrcGLzuYdT/69QWnb5Qtn76sxjgiquuWGCQuJx1/3HPyx/G2rabHaEU4pK4O8qa5SGGRknESY4nsrAlF4SwmPyeEfmRi8EDpRiAzumcdizAICvtNx1KR6V3yyJWcfEeTxdMxyemMSXrI6/M1ZP0HLTCLxf5o3IK+CHkEcVRuDc/KLRXy/pfRhfQjs8mRMiWuJnB21jUFhgqXPqdoimJkVN1Mp9jbYjj7w7Ep6rrQLqL0TUNNo1p8/w6e0VJzhwopz+i68++Hue5iaPLAyOQpScem535OL13xkx1LWqncpuRTCEkESUQMuMxQdZmZAV+biprEppR8HCzXpY8s5R9HabLenLjCyZ2Y9A2uzb4c7vfWdH5yd8kJcEkIODJ658JBEwb61p0FfeMxvOL6EU5DQmmquJ92isBxgT2vE8g4d2BPLpmVxntg13AxrqeinbRiKuuRv5yQgd311I4Mm9JE5mDT0ZSnutYypXPM8Sq7K6Id2YQgF+3O05VqcFKR+IqIey2unPbWZIoJ2KJxMR+VXCYEZHL40INPWlneCREKyFXl4uwuvWZFtg78rLuhZYuyUP46USP2gexiSKxNptMMd6MMVfd8VPG5SzV2pvqp/epKWy3bmbpSHe3bDOvaLQyF84Z1oeo1f5h9rdnpXZnd5znv0O5zOs7a1pV3Zd2dcXPvJupU9uZxvcGu8Yfaez4REG9Yf9zjzD/PIcZH4zMUjUGL+BDNj9eVbahGtV5XY9zPcpTjPp0nnJxqN1xVZlgKK/Z1CRVdAbaJhApP7l8/yB3pnR4RETsa0Ya3gF3cM2iHRUBKBlZG2aHgo6wCWa8b9D0tBWBeSkdTyeT+tgVtLuJ62tug5WAvs2/d+UPPl/jeM+JoKuXFQ3EKI44m94ovBFvMzOLIZRi3jrSxdnqDF8Ykx9ZndraCkuzbHTieb45vaVC58tVHWlBKkZcnCb9BEFM3yv7+x9fRiXbRTv+wheimSV6GGcBTf2jKPvJlokLFPtI5hiYiLFylH4QzWsRoDxgNb05w5qotB1vMFpHTrSCDp/n313obT9y15QBQlfVjEk/D2Yqo0OvQy5XDYiD6CyA/tsRo+Bc4wqSS62ZGJavBrxxHmD2YUue7xCm67QpfXclVZxW+vJq4ZZF2adeOwiWOwhgfVqK92NTXGWTeCcFk4dAMA5wEUvkA8M7okfOlalbl8B69yhH5tk0B7YsmHq1ygHHtsgUF5RUqH77T1s6TBcSWQ25b7LxDzWbgHmY2AzmGLo4/eX8IUI3OP1E+mV+LJuZr54o1sRhU7buDDlQxf/DbPGzLl701y1HR2dM6BO52brWGZzWb/bwmokwAmbQ+oMkrjgNVaf4Uha/0lIkA5ShIZuXRFMEmRnlKiP7zaewxQVR5W2Tb7Ukm0m/gIvsjTQRm9IWaTut0nXn9utN9E5TudUSrPVnwXG+nilPGs1rLluFqWXGDIyM98kRyyZW5QA9Q/arRKpuTlqsKTvM1bCQ47Vigm+C0fIkZdasmLI2H4roIrFWn+ElL7xyBNVcN63S49fc8lWBQClS2PknLhNvoXiSR250O7ajixawO5VJ8Nh3a2vM7OoXhy5kUtzPN5qre5OMqy5NFV8hpCjeVI0RUW7kz3LzIUUlXsMz9g3IhrZY5l2xhRYOrXdG+lEtsu6Ll3aVALtEwuKJV57hLS8jw9rUDNL2cntZo+s4QQK3vNlZOf5l9n6J4urrzpjsKPPDo262gqf8eAg/weiH2WKPJk0KS8kno5hCWS/37ldhzlEJkhOD5Pf2jHKRhEqEsCye8+VMYVamyOOCD2EkEpIX1t7ci2pa2nz3gtqNMSYRcKJcptZV7V4oguHJCzaDgqzmNP+ryAusxWSwJC2P1oJ3rOJbCgKKUuaY9uFWX9zDxJ0a803HnxBz1ezia0Dm3atuyvmUffs/uY6X+8dDxbPuyqTSzZgvjrmKWJ4QXSfyCP44a5FIoskcDE52FIT21DqnpnShp0aw4/unE/KfBFX5AkZB8xJZtt3NcjJDZ7QaO/e5ts47guLX7wSX0whgNpKgMT7gfmceUz5hqXbGu7CV07jo7K4cXbv8YdSn9avP53sIUE3EhT3PMAlFxcR4WlSkv9yPy39/HGU7f0DiMwpw+Oj2mMtSc5Xb+8hPJtvU1pLM1B38ZAVPV3Cip9Ry/hWkSL67XrejQilBYMlS3XWgrgkwcd+e9+8CcdTccekJwjm4G6uIANzrJUWcwtVeZ8CpUpivvNQW73RSoHLl92Pi+u+PPTahPJx2eZEN419iTP/T94zS0r2x/snbOe7S67rG/hNMYXv8NHPD8fw==&lt;/diagram&gt;&lt;/mxfile&gt;">
```

Comment:
Pls fix the spelling:
- `k8s` => `Kubernetes`
- `datadog` => `Datadog`
- `dynatrace` => `Dynatrace`
- I'd use title case for the `Telemetry Operator`
- Why do we say `Otel Collector` in the document texts, but `otel-collector` here?